coala_quickstart/Strings.py

```diff
@@ -33,3 +33,13 @@
              "all its contents but exclude your .git directory and all "
              ".o files. To do this, simply give `src/` for the first "
              "question and `.git/**,**/*.o` for the second question.")
+
+bear_docs_url = ("https://github.com/coala-analyzer/bear-docs/blob/master/"
+                 "README.rst#supported-languages")
+bear_help = ("A coala bear is a plugin that contains the checking routines. "
+             "It may have its own code or use an external linter. It "
+             "may be language specific or language independent. This "
+             "makes coala completely modularized and extensible. "
+             "Many languages including C/C++, Python, JavaScript are "
+             "supported out-of-the-box.\n\n"
+             "You can see all of them here: " + bear_docs_url)
```

Comment:
I think I'm not fully sure how you mean. Like have a file with just these strings? I mean, that'd work but involves unnecessary file I/O and stuff IMO.